src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/client/transformation/TransformationClient.java

```diff
@@ -0,0 +1,74 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.client.transformation;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.stereotype.Component;
+import org.springframework.web.client.HttpStatusCodeException;
+import org.springframework.web.client.RestTemplate;
+import org.springframework.web.util.UriComponentsBuilder;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.client.transformation.model.request.ExceptionRecord;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.client.transformation.model.response.InvalidExceptionRecordResponse;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.client.transformation.model.response.SuccessfulTransformationResponse;
+
+import static org.springframework.http.HttpStatus.BAD_REQUEST;
+import static org.springframework.http.HttpStatus.UNPROCESSABLE_ENTITY;
+
+@Component
+public class TransformationClient {
+
+    private static final Logger log = LoggerFactory.getLogger(TransformationClient.class);
+
+    private final RestTemplate restTemplate;
+
+    private final ObjectMapper objectMapper;
+
+    public TransformationClient(
+        RestTemplate restTemplate,
+        ObjectMapper objectMapper
+    ) {
+        this.restTemplate = restTemplate;
+        this.objectMapper = objectMapper;
+    }
+
+    public SuccessfulTransformationResponse transformExceptionRecord(
+        String baseUrl,
+        ExceptionRecord exceptionRecord,
+        String s2sToken
+    ) throws Exception {
```

Comment:
could be ` throws CaseTransformationException` I think

Reply:
Yes, almost. Also accompanied with `IOException`